modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/index/BasicIndexTest.java

```diff
@@ -500,6 +508,76 @@ public void testDynamicIndexesWithPersistenceIndexRebuild() throws Exception {
         }
     }
 
+    /**
+     * IGNITE-11091
+     * @throws Exception if can't start ignite node
+     */
+    @Test
+    public void testCaseSensitiveIndexNamesInGridCacheQueryManager() throws Exception {
+        inlineSize = 1;
+
+        String idxName1 = "test_v1_IDX";
+
+        String idxName2 = "test_v1_idx";
+
+        Ignite ignite = startGrids(1);
+
+        ignite.cluster().active(true);
+
+        class TestValue implements Serializable {
+            /**
+             * @param v1 Value 1.
+             */
+            private TestValue(int v1) {
+                this.v1 = v1;
+            }
+
+            @QuerySqlField
+            private final int v1;
+        }
+
+        CacheConfiguration ccfg = new CacheConfiguration()
+                .setName("testIndexesCache")
+                .setSqlSchema("PUBLIC")
+                .setAtomicityMode(CacheAtomicityMode.TRANSACTIONAL)
+                .setIndexedTypes(Integer.class, TestValue.class);
+
+        IgniteCache cache = ignite.createCache(ccfg);
+
+        cache.query(new SqlFieldsQuery("CREATE INDEX \""+idxName1+"\" on \"1TESTVALUE\" (v1)")).getAll();
+
+        cache.query(new SqlFieldsQuery("CREATE INDEX \""+idxName2+"\" on \"1TESTVALUE\" (v1)")).getAll();
+
+        GridCacheQueryManager cmgr = ((IgniteCacheProxy)cache).context().queries();
+
+        assertTrue("Index " + idxName1 + " not found in meta", indexExists(cmgr.sqlMetadata(), idxName1));
+
+        assertTrue("Index " + idxName2 + " not found in meta", indexExists(cmgr.sqlMetadata(), idxName2));
+
+        assertTrue("Index " + idxName1 + " not found in meta v2", indexExists(cmgr.sqlMetadataV2(), idxName1));
+
+        assertTrue("Index " + idxName1 + " not found in meta v2", indexExists(cmgr.sqlMetadataV2(), idxName2));
+    }
+
+    /**
+     *
+     * @param metadata metadata from grid
```

Comment:
All javadoc declarations must be started from capital letter and finishted with a dot. I.e. ` @param metadata Metadata from grid.`